distutils_cmake/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+project(pivy_cmake_setup NONE)
+cmake_minimum_required(VERSION 3.5)
+
+
+find_package(Coin CONFIG REQUIRED)
+
+if (Coin_INCLUDE_DIR)
```

Comment:
You won't reach this point if Coin cannot be found, because you added `REQUIRED` to the `find_package` call. CMake returns with a fatal error message when `REQUIRED` is given and the package cannot be found.

Reply:
because of consistency I will keep the conditional here, allthough it doesn't make any sense. But I change to Coin_FOUND.